examples/images/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width,user-scalable=no,initial-scale=1">
+    <title>gaia-fast-list/examples/images</title>
+
+    <link rel="stylesheet" href="bower_components/gaia-theme/gaia-theme.css">
+    <link rel="stylesheet" href="app.css">
+
+    <script defer src="bower_components/dom-scheduler/lib/dom-scheduler.js"></script>
+    <script defer src="bower_components/gaia-component/gaia-component.js"></script>
+    <script defer src="bower_components/poplar/poplar.js"></script>
+    <script defer src="bower_components/fast-list/fast-list.js"></script>
+    <script defer src="gaia-fast-list.js"></script>
+    <script defer src="app.js"></script>
+  </head>
+  <body class="theme-settings">
+    <gaia-fast-list top="0" bottom="0">
+      <template>
+        <li>
+          <div class="image"><img/></div>
```

Comment:
nit: even if it's more expensive I'd keep `class="image round"` for the demo :)
Looks so nice!